test/widgets/inbox_test.dart

```diff
@@ -595,6 +607,38 @@ void main() {
           check(rectAfterTap).equals(rectBeforeTap);
         });
 
+        testWidgets('shows archived label for archived streams', (tester) async {
+          final stream = eg.stream(streamId: 1, isArchived: true);
+          final subscription = eg.subscription(stream);
+          await setupPage(tester,
+            streams: [stream],
+            subscriptions: [subscription],
+            unreadMessages: [eg.streamMessage(stream: stream)]);
+          await tester.pumpAndSettle();
+
+          final headerRowFinder = findStreamHeaderRow(tester, stream.streamId);
+          check(headerRowFinder).isNotNull();
+
+          final richTextFinder = find.descendant(
+            of: find.byWidget(headerRowFinder!),
+            matching: find.byWidgetPredicate((widget) =>
+              widget is RichText &&
+              widget.text.toPlainText().contains(stream.name)
+            ),
+          );
+          expect(richTextFinder, findsOneWidget);
+
+          final archivedLabelFinder = find.descendant(
+            of: find.byWidget(headerRowFinder),
+            matching: find.text(' (archived)')
+          );
+          expect(archivedLabelFinder, findsOneWidget);
+
+          // Check that the Text widget has italic style
+          final archivedLabelText = tester.widget<Text>(archivedLabelFinder);
+          expect(archivedLabelText.style?.fontStyle, FontStyle.italic);
```

Comment:
We usually leave out checks for specific styling unless there is some extra logic to it (changing style dynamically to different inputs, etc.)